python-sdk/example_dags/example_load_file.py

```diff
@@ -159,7 +159,7 @@
     # [START load_file_example_12]
```

Comment:
This example is used in our docs to show how to use GCS's prefix but this is now completely wrong 

<img width="744" alt="image" src="https://user-images.githubusercontent.com/8811558/208516122-221831cd-7df7-4e8a-b9a6-508852a3c67a.png">